docker-compose.yml

```diff
@@ -175,7 +175,7 @@ services:
 
   next-app:
     build:
-      context: ../next-app
+      context: next-app
       dockerfile: ../docker/next-app/Dockerfile.next-app
```

Comment:
this seems like it should be `./docker...`, no?

Reply:
It's relative to the context, so it needs `..` the way it's currently set up.